pkg/sql/opt/exec/factory.opt

```diff
@@ -0,0 +1,638 @@
+# Scan returns a node that represents a scan of the given index on
```

Comment:
It'll be annoying to update, but all these comments say "returns" because they were previously functions. Maybe a quick fix of `%s/returns/represents/` would work?

Reply:
Not sure, the comments do make sense when you jump to the definition of a generated method like `ConstructScan`

Reply:
Maybe if I make the first line of the method comment` ConstructXXX returns a node that implements a XXX execution operator` and then copy the comments that explanin what the `XXX operator` means?

Reply:
Done.

Reply:
I like it 👍